src/App.css

```
@@ -1106,3 +1106,49 @@ button.augment-card {
+
+.error-boundary__message {
+  margin: 0;
+  font-weight: 600;
+}
+
+.error-boundary__detail {
+  margin: 0;
+  opacity: 0.75;
+  font-family: var(--font-mono, monospace);
+  font-size: 0.78rem;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)